etc/update.sh

```diff
@@ -14,25 +14,27 @@ cat /srv/www/www-mathe-oscar/data/webhook.secret >> .htaccess
 bundle config set --local path 'vendor/bundle'
 bundle install
 
+# use the venv defined in .venv
+# set this up before running this script!
+echo "Activating venv..."
+source .venv/bin/activate
+
+# install requirements for the python scripts
+echo "Installing python pre requisites..."
+python3 -m pip install -r etc/requirements.txt
+
 # get tutorial status
-statusurl=$(curl -SsL --retry 5 "https://api.github.com/repos/oscar-system/TutorialTesterforOscar/actions/workflows/CI.yml/runs?per_page=1" | jq  '.workflow_runs[0].jobs_url' | sed 's/"//g')
-curl -SsL --retry 5 $statusurl | jq '.["jobs"][1:] | .[] | .name+":"+.conclusion' | sed 's/^.*\s.*\s.*\s//' | sed 's/):/: /' | sed s'/"$//' > _data/examples_status.yml
+echo "Getting tutorial status....."
+./etc/tutorial_status.py
```

Comment:
To ignore the exit code, could try this:
```suggestion
./etc/tutorial_status.py || :
```